### PR TITLE
feat(project): add dashboard #166

### PR DESCRIPTION
**Pull request by ernscht**
## Purpose of this pull request? 

* Enhancement

The pull request refers to the:

* generator (generator-nitro)
* example nitro projects

## What changes did you make?

Add simple dash4 dashboard. Start it with `npm run dashboard` in your project root.
I deliberately left the feature undocumented because i'm still not sure if it works clean enough.

Let me know what you think.

### CODE REVIEW

File: packages/generator-nitro/generators/app/templates/package.json [hunk 1]
```diff
@@ -47,7 +47,7 @@
     "build:webpack": "webpack --mode production --config config/webpack/webpack.config.prod.js",<% if (options.exporter) { %>
     "export": "npm run build && gulp export",<% } %>
     "\n# RUN ------- ": "",
-    "dashboard": "dash4",
+    "dashboard": "opener http://localhost:4444 && dash4",
```
Comment (jantimon) [inline packages/generator-nitro/generators/app/templates/package.json]: `&&` causes issues on some shells - maybe use npm-run-all

Comment (ernscht) [reply packages/generator-nitro/generators/app/templates/package.json]: What issues are you thinking about?
We have been using && in other scripts for years (see line before) without negative feedback.

Comment (jantimon) [reply packages/generator-nitro/generators/app/templates/package.json]: Hey @ernscht 

you are right sorry for the false alarm :)  
Not sure if this was fixed in the meantime but it works very well now for me even if my terminal does not support it:

and_and

File: packages/generator-nitro/generators/app/templates/dash4.config.js [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+						new PluginTerminal({
+							title: 'Cypress Mode',
+							cmd: 'npm run cypress-test',
+							dark: true,
+							autostart: false,
+						}),
+					],
+					[
+						new PluginNpmScripts({
+							scripts: [
```
Comment (janwidmer) [inline packages/generator-nitro/generators/app/templates/dash4.config.js]: Is it possible to define a group title like with `PluginActions`?

**Comment by janwidmer** [comment]
General Thoughts:

It works great so far..I just asked my self if I really want to have the terminal running in the browser as chrome is eating all of my ram already without running a terminal in the browser.. But of course I can remove that for my project..

File: packages/generator-nitro/generators/app/templates/dash4.config.js (hunk 2 above)
Comment (ernscht) [reply packages/generator-nitro/generators/app/templates/dash4.config.js]: no. 
What would you write down as a title?

**Comment by ernscht** [comment]
> I just asked my self if I really want to have the terminal running in the browser as chrome is eating all of my ram already without running a terminal in the browser.. 

RAM is not really expensive and you're saving some RAM from the terminal. ;-)

> But of course I can remove that for my project..

... maybe it's not for you. This should give an overview of possibilities for users without deeper knowledge of the project.

File: packages/generator-nitro/generators/app/templates/dash4.config.js (hunk 2 above)
Comment (ernscht) [reply packages/generator-nitro/generators/app/templates/dash4.config.js]: let's make a feature request

**Pull request merged**